daisy_workflows/build-publish/debian/debian_10.wf.json

```diff
@@ -19,6 +19,14 @@
     "gcs_url": {
       "Required": true,
       "Description": "The GCS path that image raw file exported to."
+    },
+    "run_sbom_bool": {
+      "Value": "false",
+      "Description": "Determines if the SBOM for the image will be generated"
```

Comment:
for consistency please finish descriptions with dot (just like `gcs_url` does).

Reply:
done in other pr